framework/src/modules/chain/block_processor_v2.js

```diff
@@ -224,6 +224,13 @@ class BlockProcessorV2 extends BaseBlockProcessor {
 				return this.bftModule.computeBFTHeaderProperties(delegatePublicKey);
 			},
 			(data, bftHeader) => this._create({ ...data, ...bftHeader }),
+			async (data, block) => {
```

Comment:
Can you add a comment above the create.pipe briefly explaining the three steps that are happening in the create pipeline?